cfgov/jinja2/v1/_includes/organisms/_vars-mega-menu.html

```diff
@@ -66,110 +73,256 @@
-            ('Rules Under Development', '/policy-compliance/rulemaking/rules-under-development/', []),
-            ('Regulatory Agenda', '/policy-compliance/rulemaking/regulatory-agenda/', []),
-            ('Small Business Review Panels', '/policy-compliance/rulemaking/small-business-review-panels/', []),
-        )]),
-    ),(
-        ('Compliance & Guidance', '/policy-compliance/guidance/', [(
-            ('Implementation & Guidance', '/policy-compliance/guidance/implementation-guidance/', []),
-            ('Supervision & Examinations', '/policy-compliance/guidance/supervision-examinations/', []),
-            ('Supervisory Highlights', '/policy-compliance/guidance/supervisory-highlights/', [])
-        )]),
-        ('Enforcement', '/policy-compliance/enforcement/', [(
-            ('Enforcement Actions', '/policy-compliance/enforcement/actions/', []),
-            ('Petitions to Modify or Set Aside', '/policy-compliance/enforcement/petitions/', []),
-            ('Warning Letters', '/policy-compliance/enforcement/warning-letters/', [])
-        )])
-    ),(
-        ('Notices & Opportunities to Comment', '/policy-compliance/notice-opportunities-comment/', [(
-            ('Open Notices', '/policy-compliance/notice-opportunities-comment/open-notices/', []),
-            ('Archive of Closed Notices', '/policy-compliance/notice-opportunities-comment/archive-closed/', [])
-        )]),
-        ('Amicus Program', '/policy-compliance/amicus/', [(
-            ('Filed Briefs', '/policy-compliance/amicus/briefs/', []),
-            ('Suggest a Case', '/policy-compliance/amicus/suggest/', [])
-        )])
-    )]),
-    ('About Us', '/about-us/', [(
-        ('The Bureau', '/about-us/the-bureau/', []),
-        ('Budget & Strategy', '/about-us/budget-strategy/', []),
-        ('Payments to Harmed Consumers',
-         '/about-us/payments-harmed-consumers/', [])
-    ),(
-        ('Blog','/about-us/blog/', []),
-        ('Newsroom', '/about-us/newsroom/', []),
-        ('Events', '/about-us/events/', []),
-        ('Recent Postings', '/activity-log/', [])
-    ),(
-        ('Careers', '/about-us/careers/', [
-            (
-                ('Working @ CFPB', '/about-us/careers/working-at-cfpb/', []),
-                ('Job Application Process', '/about-us/careers/application-process/', []),
-                ('Students & Recent Graduates', '/about-us/careers/students-and-graduates/', []),
-                ('All Current Openings', '/about-us/careers/current-openings/', [])
-            )
-        ]),
-        ('Doing Business With Us', '/about-us/doing-business-with-us/', []),
-        ('Advisory Groups', '/about-us/advisory-groups/', []),
-        ('Project Catalyst', '/about-us/project-catalyst/', []),
-        ('Contact Us', '/about-us/contact-us/', [])
-    )])
-)] -%}
+
+{% set consumer_tools_group_one =  {
+    'title': 'Money Topics', 
+    'nav_items': [
+        {'caption': 'Auto Loans','url': '/consumer-tools/auto-loans-topic/'} ,
```

Comment:
That makes sense to me! Updated in https://github.com/cfpb/cfgov-refresh/pull/3147/commits/194f5f2c31c5342a26fe78c7342602083a7681a4.